### DISCUSSION TITLE
Delete GitHub Projects

### DISCUSSION THREAD

**Discussion by hrishikesh-k** [community/community#23613] [New to GitHub]
<p>Hello,</p>
<p>I created a project under my profile. The project is not associated with any repo and it’s simply visible here:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/Hrishikesh-K?tab=projects&amp;type=beta">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/Hrishikesh-K?tab=projects&amp;type=beta" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/11403551/181078227-a9476e45-4954-4e87-b7cc-a5bd6f185cbf.png" class="thumbnail onebox-avatar" width="460" height="460">

<h3><a href="https://github.com/Hrishikesh-K?tab=projects&amp;type=beta" target="_blank" rel="noopener nofollow ugc">Hrishikesh-K - Projects</a></h3>

  <p>👨‍💻. Hrishikesh-K has 6 repositories available. Follow their code on GitHub.</p>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>I wish to delete this project, but I can’t seem to find any option to do so. I can close it and set it to private, but if I could completely delete it, that would be awesome.</p>
<p>Could someone guide me on if that’s possible?</p>

**Comment by semenkadi** [comment]
<p>не понимаю. все же упоминается в введении. шпаргалка есть же.</p>

**Comment by hrishikesh-k** [comment, marked as answer]
<p>Nevermind, I found this:</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.com/github/feedback/discussions?discussions_q=delete%2Bproject">
  <header class="source">
      <img src="https://github.githubassets.com/favicons/favicon.svg" class="site-icon" width="32" height="32">

      <a href="https://github.com/github/feedback/discussions?discussions_q=delete%2Bproject" target="_blank" rel="noopener nofollow ugc">GitHub</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/11403551/181078229-a333fed3-0c96-441c-b5d8-04c61dc894fe.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3acbf3d42511b5a353d42d3abc61ebd8fce4f43d_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3acbf3d42511b5a353d42d3abc61ebd8fce4f43d_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/3/a/3acbf3d42511b5a353d42d3abc61ebd8fce4f43d.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/3/a/3acbf3d42511b5a353d42d3abc61ebd8fce4f43d_2_10x10.png"></div>

<h3><a href="https://github.com/github/feedback/discussions?discussions_q=delete%2Bproject" target="_blank" rel="noopener nofollow ugc">Discussions · github/feedback</a></h3>

  <p>Public feedback discussions for: GitHub for Mobile, GitHub Discussions, GitHub Codespaces, GitHub Sponsors, GitHub Issues and more! - Discussions · github/feedback</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<p>where I can see a lot of people requesting this already.</p>
